stan/math/prim/fun/constants.hpp

```diff
@@ -26,6 +26,12 @@ inline double e() { return boost::math::constants::e<double>(); }
  */
 inline double pi() { return boost::math::constants::pi<double>(); }
 
+/**
+ * The value of 10000000000, ie a big integer.
```

Comment:
This constant should be local to the function that uses it.  Or at the very least put in the internal namespace.

If it does get documented somewhere, "ie" should be "i.e.," with two periods and a comma.  It should also mention why this value was chosen.

Reply:
I've removed this constant from the constants file and put it in the internal namespace.

Interestingly, using INT_MAX fails the floating point accuracy. I did a loop from 1e10 to INT_MAX to find the biggest INT that doesn't fail. It's 2e10. That is added to the documentation of the functions.